bin/phpunit-docker.sh

```diff
@@ -1,15 +1,51 @@
 #!/bin/bash
 
 # Note: you can pass in additional phpunit args
-# Test a specific file: ./bin/phpunit-docker.sh tests/path/to/test.php
+# Test with explicit WP version and Multisite config: ./bin/phpunit-docker.sh --version 5.4.4 --multisite 0
+# Test with explicit WP version and file: ./bin/phpunit-docker.sh --file tests/path/to/test --version 5.4.4
+# Test a specific file: ./bin/phpunit-docker.sh --file tests/path/to/test.php
 # Stop on failures: ./bin/phpunit-docker.sh --stop-on-failure
 # etc.
 
-WP_VERSION=${2-latest}
-WP_MULTISITE=${3-0}
+while test $# -gt 0; do
+  case "$1" in
+  --version)
```

Comment:
Thoughts on using `--wp` instead since `--version` is more commonly used to find out the version of a given script?

Reply:
Other than that, this looks great 👍  Named params are so much nicer to use :)